clearpath_sensors/launch/flir_blackfly.launch.py

```diff
@@ -49,14 +51,20 @@ def generate_launch_description():
     arg_param_mapping_file = DeclareLaunchArgument(
         'param_mapping_file',
         default_value=PathJoinSubstitution([
-            FindPackageShare('spinnaker_camera_driver'), 'config',
-             'blackfly_s.yaml'
+            FindPackageShare('spinnaker_camera_driver'),
+            'config',
+            'blackfly_s.yaml'
         ]))
 
     arg_namespace = DeclareLaunchArgument(
         'namespace',
         default_value='platform/sensors/camera_0')
```

Comment:
This should be `sensors/camera_0` based on https://docs.clearpathrobotics.com/docs/ros/api/sensors_api

Reply:
Done